src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -681,6 +682,9 @@
         "RESTRICT",
         Sequence("NO", "ACTION"),
     ),
+    UnknownLiteralSegment=StringParser(
+        "unknown", LiteralKeywordSegment, type="null_literal"
```

Comment:
```suggestion
        "UNKNOWN", LiteralKeywordSegment, type="null_literal"
```
nit - we normally uppercase the strings